library/alloc/src/collections/linked_list.rs

```diff
@@ -53,8 +53,7 @@ struct Node<T> {
 /// This `struct` is created by the [`iter`] method on [`LinkedList`]. See its
```

Comment:
If you refer to it by its path, you don't need a separate link at the bottom ;)
```suggestion
/// This `struct` is created by [`LinkedList::iter()`]. See its
```